Assignments/SCF_Session-2/SCF_Assignment-1/src/HexCalculatorImplementation.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Scanner;
+
+public class HexCalculatorImplementation {
+	public static void main(String args[]){
+		Scanner sc=new Scanner(System.in);
+		String firstNumber="";
+		String secondNumber="";
+		String data="";
+		System.out.println("Enter you choice:");
+		System.out.println("1.Add  2.Subtract  3.Multiply  4.Divide  5.check equals  6.check greater than  7.check less than  8.convert into Decimal  9.conver into hex");
+		int menuChoice=sc.nextInt();
+		if(menuChoice>=1 && menuChoice<=7){
+			System.out.println("Enter the First Number: ");
+			firstNumber=sc.next();
+			System.out.println("Enter the Second Number: ");
+			secondNumber=sc.next();
+
+		}
+		else{
+			System.out.println("Enter the data: ");
+			data=sc.next();
+
+		}
+		HexCalculator obj=new HexCalculator(); //object declaration of HexCalcultor class
+		switch(menuChoice){
+			
+		case 1:
+			System.out.println(obj.add(firstNumber, secondNumber));
```

Comment:
@meta-vaibhav-gautam While showing output, you have to make sure what you are displaying. Just like Sum of numbers or Subtract of numbers etc.